util/test/checkCopyrights.bash

```diff
@@ -18,6 +18,14 @@
 #  Make*
 #
 
+todate=$(date "+%m%d")
+cond=$(date -d 01-07 +"%m%d")
+
+if [ $todate -le $cond ];
+then
+ exit 0;
+fi
```

Comment:
I put this snippet in a .bash file and added a printing line afterwards to see if I got through.  On linux, it exited without printing the line at the end (so I think this is incorrect?) and on my Mac I got
```
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
./foo.bash: line 4: [: 1222: unary operator expected
```
followed by the line I was expecting to print

Reply:
`-le` is integer comparison right?
I would think something like:
```bash
[[ $(date "+%m-%d") < '01-07' ]] && exit 0
```